### DISCUSSION TITLE
Eject sidebar_component

### DISCUSSION THREAD

**Discussion by sageworksstudio** [avo-hq/avo#3438] [Help] [answered]
I'm trying to eject the sidebar component, but this `rails g avo:eject --component app/components/avo/sidebar_component` Results in Failed to find the `app/components/avo/sidebar_component` component

What is the correct path to eject that component?

**Comment by Paul-Bob** [comment, marked as answer]
Hi @sageworksstudio no need to include `app/components` in the path since that's the root directory where the ejector starts looking.

![image](https://github.com/user-attachments/assets/f6c4f7d0-fdcd-4a2b-9f7e-5fafdfe6cf5b)

This will work as expected: `rails g avo:eject --component avo/sidebar_component`

Or: `rails g avo:eject --component Avo::SidebarComponent`

**Comment by sageworksstudio** [comment]
Thank you!
